crates/console/src/ForgeConsole.json

```diff
@@ -0,0 +1,366 @@
+[
```

Comment:
Could you minimize it so it is in one line like `Console.json` to avoid bloating git diff?